src/monattack.cpp

```diff
@@ -720,6 +747,9 @@ void mattack::vine(monster *z, int index)
 void mattack::spit_sap(monster *z, int index)
 {
     // TODO: Friendly biollantes?
+    if( z->friendly ) {
+        return;
```

Comment:
Possible if you assimilate a triffid heart.  As such, I'd think 'em an organic turret and just change up the targeting.